IBPSA/Resources/Scripts/Dymola/Fluid/Sources/Examples/MassFlowSource_WeatherData.mos

```diff
@@ -1,3 +1,4 @@
+Advanced.StoreProtectedVariables = true;
```

Comment:
Why is this needed, and if it is needed, why don't you set it back to its old value after the simulation and plotting?

Reply:
The class `sin_with_h` extends `IBPSA.Fluid.Sources.BaseClasses.PartialSource`, in which `medium` is protected.